build.gradle

```diff
@@ -66,6 +66,7 @@ dependencies {
     //GRPC:
     implementation "io.grpc:grpc-protobuf:${grpcVersion}"
     implementation "io.grpc:grpc-stub:${grpcVersion}"
+    testImplementation 'junit:junit:4.13.1'
```

Comment:
The project already has tests why is this needed?

Reply:
to function